packages/@okta/vuepress-site/guides/sign-into-spa/require-authentication/vue/reqauthspecific.md

```diff
@@ -0,0 +1,30 @@
+Set the `requiresAuth` property to `true` in the optional `meta` object as part of the route defintion.
```

Comment:
Set the `requiresAuth` property to `true` in the optional `meta` object as part of the route definition.